README.md

```diff
@@ -10,3 +10,15 @@ $ sbt release
 
 will publish to [Maven Central](http://search.maven.org/) via Sonatype. You will need Sonatype credentials and a PGP key. It can take up to 2hrs to show up in search.
 
+## Information about built jars
+
+The content-api-models project builds the following jar files: 
+
+* content-api-models-scala - Scrooge generated class files based on the Thrift definitions of the content api models found in the `content-api-models` dependency. 
+
+* content-api-models-json - Json parsers and deserializers. Used internall by the content api to convert from Elasticsearch returned json to the Scrooge-generated Thrift classes. As a client you should never need to depend on this.
```

Comment:
It's also used by the Scala client, so it's not only internal to Concierge. In other words, clients might depend on it, but probably only transitively.